src/client/lib/consentsTracking.ts

```diff
@@ -63,6 +63,18 @@ const newslettersFormSubmitOphanTracking = (
       }
     });
   }
+
+  // @AB_TEST: Default Weekly Newsletter Test: required for tracking
+  inputElems.forEach((elem) => {
+    const defaultWeeklyNewsletter = elem.name === '6028';
+    if (defaultWeeklyNewsletter) {
+      trackInputElementInteraction(
+        elem,
+        'newsletter',
+        'saturday-roundup-trial',
```

Comment:
I've hardcoded this for the trial, as adding it to page data makes it show up as a regular newsletter card.
Open to more graceful solutions and will find something better if we decide to roll this out as permanent change - this test will be rolled back regardless the outcome.